pkg/cmd/server/origin/master.go

```diff
@@ -261,8 +150,6 @@ func (c *MasterConfig) Run(kc *kubernetes.MasterConfig, assetConfig *AssetConfig
 		extra []string
```

Comment:
That `extra` variable name is like from hell, I'd vote to rename it to something meaningful, like messages, logs, etc.